WcaOnRails/app/views/registrations/_register_form.html.erb

```diff
@@ -12,7 +12,8 @@
   <%= f.input :comments, disabled: disabled %>
 
   <%# If the user has to pay fee here, this information will already be in the payment form %>
-  <% unless @registration.has_to_pay_fee_here? %>
+  <%# If there are no entry fees, no need to diplay this information %>
+  <% unless @registration.has_to_pay_fee_here? || @registration.entry_fee == 0 %>
```

Comment:
1: fixed for something clearer.

2: it's actually indirectly in since it uses `outstanding_entry_fees`.
The thing is that we want to show this form "input" if there is any entry fee, but not if the payment form is showing (because the input is already there and we don't need to duplicate the information).